client/src/components/SendMessage.js

```diff
@@ -1,14 +1,17 @@
 'use strict';
 
 import React from 'react';
+import EmojiPicker from './EmojiPicker'
 import Actions from "actions/UIActions";
 import 'styles/SendMessage.scss';
 
 class SendMessage extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      theme: props.theme
+      theme: props.theme,
+      useEmojis: props.useEmojis,
+      emojiPickerActive: false
     };
```

Comment:
Let's add a another variable to the state here: `lastWord: null`. See below for more details as to why.